packages/react/src/components/Popup/Popup.tsx

```diff
@@ -414,42 +433,34 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
     accessibility: ReactAccessibilityBehavior,
     { placement, scheduleUpdate }: PopperChildrenProps,
   ) => {
-    const { content: propsContent, renderContent, contentRef, mountDocument, pointing } = this.props
+    const {
+      content: propsContent,
+      renderContent,
+      contentRef,
+      mountDocument,
+      pointing,
+      trapFocus,
+      autoFocus,
+    } = this.props
     const content = renderContent ? renderContent(scheduleUpdate) : propsContent
     const documentRef = toRefObject(mountDocument)
 
-    const popupWrapperAttributes = {
+    const popupContentAttributes = {
```

Comment:
It seems that `popupContentAttributes` is useless now and can be merged with `popupContent`:

```jsx
    const popupContent = Popup.Content.create(content, {
      defaultProps: {
        ...(rtl && { dir: 'rtl' }),
        ...accessibility.attributes.popup,
        ...accessibility.keyHandlers.popup,
        className: popupPositionClasses,
        ...this.getContentProps(),
        placement,
        pointing,
        pointerRef: this.pointerTargetRef,
        trapFocus,
        autoFocus,
      },
      overrideProps: this.getContentProps,
    })
```